upgrades/delete_dashboards.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+
+import ConfigParser
+import requests
+
+from requests.auth import HTTPBasicAuth
+
+
+def delete_dashboards(server_ip, user, password):
+
+    dashboards = ["cluster-dashboard", "brick-dashboard", "host-dashboard",
+                  "volume-dashboard"]
+
+    headers = {'content-type': 'application/json'}
+
+    for dashboard in dashboards:
+        url = "http://%s:3000/api/dashboards/db/%s" % (server_ip, dashboard)
+        print (url)
+        response = requests.delete(url, headers=headers,
+                                   auth=HTTPBasicAuth(user, password))
+        if response.status_code == 200:
+            print "Deleted", dashboard, "\n "
+        else:
+            print "Failed to delete", dashboard, response.json(), "\n"
+
+
+    # Deleting the alerts dashboards
+    url = "http://%s:3000/api/orgs/name/Alert_dashboard" \
+          % server_ip
+    print "Getting alerts organization id\n",url,"\n"
+    response = requests.get(url, headers=headers,
+                            auth=HTTPBasicAuth(user, password))
+    resp = response.json()
+
+    if 'id' in resp:
+        id = resp['id']
+        url = "http://%s:3000/api/orgs/%s" % (server_ip, id)
+        print "Deleting alerts organization\n", url
+        response = requests.delete(url, headers=headers,
+                                   auth=HTTPBasicAuth(user, password))
+        resp = response.json()
+        if resp == {u'message': u'Organization deleted'}:
+            print "Deleted Alert dashboards"
+        else:
+            print "Failed to delete Alert dashboards ", resp
+
+    else:
+        print "Failed to delete Alert dashboards."
+        print "Organization id not found", resp
+
+
+def main():
+
+    # getting grafana admin_username and password
+    config = ConfigParser.ConfigParser()
+    config.read('/etc/tendrl/monitoring-integration/grafana/grafana.ini')
+    default_user = config.get('security', 'admin_user')
+    default_password = config.get('security', 'admin_password')
+    default_ip = "127.0.0.1"
+
+    username = raw_input("Please enter the grafana admin_username "
+                         "default-[%s]: " % default_user) or default_user
+    password = raw_input("Please enter the grafana admin_password default"
```

Comment:
Would you be able to hide this while user enters the password? That would be better.

Reply:
@shtripat yes, it can be hidden, I thought of showing it so that user can cross-verify it.

Reply:
I am fine with any approach as this is just a tool, but personally I feel it should be hidden. Anyway if wrong password passed, later stage it would fail with error.

Reply:
@shtripat Please review, I added command line args
It will work like: python test.py --username=admin --password=pass

If the args are not supplied, it will take from grafana.ini